### DISCUSSION TITLE
Pork Enemy

### DISCUSSION THREAD

**Discussion by Jumalita** [2WolfGames/Erlang-Legacy#188] [Ideas]
This is the behaviour that I think pork enemy can have.

**Entrance**
I think Pork should fall from the roof as, at the same time, two big rocks fall to delemiter combat space.
While this happend ajax will be frozen.
The battle will start when Pork screams. 

**First fase**
_Ajax will have all the powers?_
Pork cheases Ajax from one way to another. (without jumping so Player can dodge it)
The chase ends when pork hits the wall and remains a few seconds immobile. This will be the safest time to attack him.

**Second Fase**
Pork goes to the center of the ring and jumps few times. This will make few rocks fall from ceiling, rocks that Ajax will need to dodge.

In this fase Pork will chease ajax as the first fase and also will kick the walls with his paws to invoke failing rocks. 

**Third Fase** 
As second fase we show the player that he has started a new face when pork goes to the center of the ring and jumps few times to make some rocks fall.

In this fase Pork will chease ajax as the first and second fase and also will jump to the place where ajax is. 
_If we have the dash this jump can be faster, otherwishe it has to be slower... let's discuss it in the comments_

**Death**
The pork makes a huge scream and runs across the ring breaking the walls that delemited space. 

_Maybe i have an idea for the story that could work for all the enemies of the first island_

**Comment by Jumalita** [comment]
After fase 1 compleated, the pork doesn't have to hit the wall and remain a few seconds inconcient every time that chases ajax

**Reply by Jumalita** [reply]
Shut up.

**Reply by wilberquito** [reply]
@Jumalita we dont have scream animation

**Reply by Jumalita** [reply]
We can use pre-attack animation +  a pig scream sound 
Do not need more.

**Reply by wilberquito** [reply]
@Jumalita we definetly need a screen animation beacuse pre attack finishes with idle position
